Lib/test/test_pathlib.py

```diff
@@ -37,7 +37,7 @@ def with_segments(self, *pathsegments):
         return type(self)(*pathsegments, session_id=self.session_id)
 
 
-class _BasePurePathTest(object):
+class _BaseBasePurePathTest(object):
```

Comment:
This naming is very silly. It would be good to land #104829 first, which makes the test class hierarchy roughly match the pathlib class hierarchy as it is today, rather than as it was in the very early days of pathlib! This PR would change the class name here from `PurePathTest` to `BasePurePathTest`, which is much more reasonable.